standalone-metastore/metastore-server/src/main/sql/mysql/upgrade-4.0.0-alpha-1-to-4.0.0-alpha-2.mysql.sql

```diff
@@ -0,0 +1,240 @@
+SELECT 'Upgrading MetaStore schema from 4.0.0-alpha-1 to 4.0.0-alpha-2' AS MESSAGE;
```

Comment:
Shouldn't this MySQL upgrade script be mostly empty (except version update) at the moment?

Reply:
Thanks for catching this.
Fixed